### PR TITLE
Hotfix/rax40 include kernel modules

### PR DESCRIPTION
**Pull request by rmelotte**
- Fix for https://github.com/prplfoundation/prplMesh/issues/1222 (remove the `tmp` folder before generating the full defconfig
- Save the full .config file in the artifacts

### CODE REVIEW

File: tools/docker/builder/openwrt/Dockerfile [hunk 1]
```diff
@@ -92,7 +92,8 @@ RUN cp feeds.conf.default feeds.conf \
     && if [ "$TARGET_PROFILE" = "DEVICE_NETGEAR_RAX40" ] ; then \
         grep -v -E 'CONFIG_SDK|CONFIG_MAKE_TOOLCHAIN' feeds/intel/rax40.profile >> .config ;\
     fi ;\
-    make defconfig \
+    rm -rf tmp \
```
Comment (arnout) [inline tools/docker/builder/openwrt/Dockerfile:97]: Adding an explanatory comment here would be nice. A little trick to add a comment:
```suggestion
    : "Installing intel feed doesn't correctly regenerate kernel .package-info"; \
    : "force regeneration by removing it"; \
    rm -rf tmp \
```

File: tools/docker/builder/openwrt/scripts/build.sh [hunk 2]
```diff
@@ -16,3 +16,4 @@ PRPLMESH_VERSION=${PRPLMESH_VERSION}
 EOT
 find bin -name 'prplmesh*.ipk' -exec cp -v {} "artifacts/prplmesh.ipk" \;
 find bin/targets/"$TARGET_SYSTEM"/"$SUBTARGET"/ -type f -maxdepth 1 -exec cp -v {} "artifacts/" \;
+cp .config artifacts/
```
Comment (arnout) [inline tools/docker/builder/openwrt/scripts/build.sh]: Does gitlab-ci render "hidden" files nicely? If not, we may have to rename it, e.g. to `openwrt.config`.

**Review by arnout — APPROVED** [review]
Nitpicks only.

**Review by tomereli — APPROVED** [review]

File: tools/docker/builder/openwrt/Dockerfile (hunk 1 above)
Comment (rmelotte) [reply tools/docker/builder/openwrt/Dockerfile:97]: Nice! There were multiple other occasions where I wanted to put comments in the middle like here, but I couldn't think of any way to do it.

I'll do this one here and remember it for the next time.

Thanks!

File: tools/docker/builder/openwrt/scripts/build.sh (hunk 2 above)
Comment (rmelotte) [reply tools/docker/builder/openwrt/scripts/build.sh]: I don't know how Gitlab renders them, but I applied your suggestion anyway.

**Pull request merged**